apps/expo/src/forms/CreateTestForm/index.tsx

```diff
@@ -613,62 +613,61 @@ const CreateTestForm: FC<Props> = ({
+                >
+                  <Text className="font-nunito-bold w-70 text-right text-lg font-semibold leading-6 text-[#6949FF]">
+                    View All
                   </Text>
-                  <TouchableOpacity
-                    className="flex flex-row items-center gap-1"
-                    onPress={goToViewAllQuestions}
-                  >
-                    <Text className="font-nunito-bold w-70 text-right text-lg font-semibold leading-6 text-[#6949FF]">
-                      View All
-                    </Text>
-                    <RightArrowIcon />
-                  </TouchableOpacity>
-                </View>
+                  <RightArrowIcon />
+                </TouchableOpacity>
+              </View>
+            )}
 
-                <SafeAreaView className="min-h-full flex-1">
-                  {isLoading ? (
-                    <View className="h-[50%] w-full flex-col justify-between self-center">
-                      <View className="my-7">
-                        <SkeletonLoader
-                          isCircular={true}
-                          width={"100%"}
-                          height={75}
-                        />
-                      </View>
-                      <View className="my-7">
-                        <SkeletonLoader
-                          isCircular={true}
-                          width={"100%"}
-                          height={75}
-                        />
-                      </View>
-                    </View>
-                  ) : (
-                    <FlashList
-                      estimatedItemSize={10}
-                      data={readyQuestions}
-                      showsVerticalScrollIndicator={true}
-                      renderItem={({ item: question, index }) => {
-                        return (
-                          <QuestionCard
-                            question={question}
-                            index={index}
-                            goToEditQuestion={goToEditQuestion}
-                          />
-                        );
-                      }}
+            <SafeAreaView className="min-h-full flex-1">
+              {isLoading ? (
+                <View className="h-[50%] w-full flex-col justify-between self-center">
+                  <View className="my-7">
+                    <SkeletonLoader
+                      isCircular={true}
+                      width={"100%"}
+                      height={75}
                     />
-                  )}
-                </SafeAreaView>
-              </View>
-            </>
-          )}
+                  </View>
+                  <View className="my-7">
+                    <SkeletonLoader
+                      isCircular={true}
+                      width={"100%"}
+                      height={75}
+                    />
+                  </View>
+                </View>
+              ) : (
+                <FlashList
```

Comment:
Returns warning that says "FlashList's rendered size is not usable. Please make sure that the parent view of the list has a valid size."